internal/app/wwctl/container/exec/child/main.go

```
@@ -34,18 +36,21 @@ func CobraRunE(cmd *cobra.Command, args []string) (err error) {
 		os.Exit(1)
 	}
 	conf := warewulfconf.Get()
+	if overlayDir == "" {
+		overlayDir = path.Join(conf.Paths.WWChrootdir, "overlays")
+	}
 	mountPts := conf.MountsContainer
 	mountPts = append(container.InitMountPnts(binds), mountPts...)
 	// check for valid mount points
 	lowerObjects := checkMountPoints(containerName, mountPts)
-	overlayDir := conf.Paths.WWChrootdir + "/overlays"
 	// need to create a overlay, where the lower layer contains
 	// the missing mount points
 	wwlog.Verbose("for ephermal mount use tempdir %s", overlayDir)
 	// ignore errors as we are doomed if a tmp dir couldn't be written
 	_ = os.MkdirAll(path.Join(overlayDir, "work"), os.ModePerm)
 	_ = os.MkdirAll(path.Join(overlayDir, "lower"), os.ModePerm)
 	_ = os.MkdirAll(path.Join(overlayDir, "nodeoverlay"), os.ModePerm)
```

If we're doomed, maybe we shouldn't ignore the errors.